### PR TITLE
Update release notes with several recent changes

### PR DESCRIPTION
**Pull request by solarissmoke**

### CODE REVIEW

File: docs/source/releases/v1.6.rst [hunk 1]
```diff
@@ -19,10 +19,10 @@ Table of contents:
 Compatibility
 -------------
 
-Oscar 1.6 is compatible with Django 1.8, 1.10 and 1.11 as well as Python 2.7,
-3.3, 3.4, 3.5 and 3.6. Support for Django 1.9 is no longer officialy supported
-since it is longer supported by Django (end of life).
+Oscar 1.6 is compatible with Django 1.11 and 2.0 as well as Python 2.7,
+3.4, 3.5 and 3.6.
 
+Official support for Django 1.8, 1.9 and 1.10 has been dropped.
```
Comment (sasha0) [inline docs/source/releases/v1.6.rst]: Hm, not true yet for tox.

File: docs/source/releases/v1.6.rst [hunk 2]
```diff
@@ -32,36 +32,41 @@ What's new in Oscar 1.6?
...
 ~~~~~~~~~~~~~
- - Majority of the Oscar imports now using dynamic loading, instead of
-   direct importing for better extensibility and customizability, except
-   signals.
- - Customer transaction emails reworked on top of mailgun layout and has
-   better look.
+ - The majority of the Oscar class imports now use dynamic loading, instead of
+   direct importing, for better extensibility and customizability.
+
+ - Basic HTML email templates added for customer transaction emails.
```
Comment (sasha0) [inline docs/source/releases/v1.6.rst]: They are not quite basic - since have original styling.

File: docs/source/releases/v1.6.rst [hunk 3]
```diff
@@ -32,36 +32,41 @@ What's new in Oscar 1.6?
...
- - Product attribute value save now handled by separate methods, similar to
-   validation which allows to override save logic for file, multi-option or
-   generic attribute values.
+
+ - The default shipping method ( ``oscar.apps.basket.views.BasketView.get_default_shipping_address``)
+   on basket summary page now uses the user's default shipping address to
+   calculate an indicative shipping cost.
+
+ - Fixed `ExtendedURLValidator` for locale-prefixed URLs, for locales different
+   from the current locale (see :issue:`1481`).
```
Comment (sasha0) [inline docs/source/releases/v1.6.rst]: Not sure why detailed explanation was removed.

File: docs/source/releases/v1.6.rst (hunk 1 above)
Comment (solarissmoke) [reply docs/source/releases/v1.6.rst]: Yes, but these are release notes for 1.6, which will when we release it drop support for Django 1.10 and below?

File: docs/source/releases/v1.6.rst (hunk 3 above)
Comment (solarissmoke) [reply docs/source/releases/v1.6.rst]: You have linked to the associated ticket - I think that is sufficient.

File: docs/source/releases/v1.6.rst (hunk 1 above)
Comment (sasha0) [reply docs/source/releases/v1.6.rst]: Yes, but we haven't yet - let's firstly drop it and then add note.

**Comment by solarissmoke** [comment]
Closing in favour of #2508.

**Pull request closed without merging**